torch/lib/c10d/reducer.cpp

```diff
@@ -1008,6 +996,31 @@ void Reducer::prepare_for_backward(
         "has flow control causing later iterations to have unused parameters.");
   }
 }
+// Traverse the autograd graph starting at the specified output.
```

Comment:
Should this comment go before search_unused_parameters function now? And leave a more general comment about prepare for backward here?